lib/wraft_doc/search/typesense.ex

```diff
@@ -0,0 +1,34 @@
+defmodule WraftDoc.Search.Typesense do
+  @moduledoc """
+  Handles operations for Typesense
+  """
+
+  alias WraftDoc.Search.Encoder
+
+  def create_collection(schema) do
+    ExTypesense.create_collection(Encoder.to_document(schema))
```

Comment:
To create a collection what we need is the schema, not the document. I think we already discussed this in the last session.